server/util/timeutil/timeutil.go

```diff
@@ -3,8 +3,29 @@ package timeutil
 import (
 	"fmt"
 	"time"
+
+	"google.golang.org/protobuf/types/known/durationpb"
+	"google.golang.org/protobuf/types/known/timestamppb"
 )
 
+// Returns time.Time from a timestamp proto field. When the given ts is nil, uses fallbackMillis
+func GetTime(ts *timestamppb.Timestamp, fallbackMillis int64) time.Time {
```

Comment:
nit: I hate to advocate for longer names, but I think calling these GetTimeWithFallback and GetDurationWithFallback might make the code clearer at the callsites. WDYT?

Reply:
Done